usagov/vote-gov-nvrf-app
# Vote-503: Check Box error for eligibility page

## Jira ticket

[Vote-503](https://cm-jira.usa.gov/browse/VOTE-503)

## Description

This PR will address the error on the checkbox for the eligibility page.  The bug was when moving forward if the user did not click the checkbox the message would appear however, when it was selected the message wouldn't disappear. This will fix that error by using the same function on the confirmation page. 

[Preview Link](https://federalist-aef5b597-8e18-44b6-aeba-3fc3f17cdac1.sites.pages.cloud.gov/preview/usagov/vote-gov-nvrf-app/bug/vote-503-check-box/)

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. visit the app with the preview link above and work your way to the eligibility page... once you are there try and navigate to the next page you should be prompted with the error message to confirm eligibility, the expected behavior is when the checkbox is clicked message should go away.
2. use the form back button to change your state option, and navigate your way back to the eligibility page, The expected behavior should be that the checkbox is clear and you will need to reconfirm.

## Checklist for the Developer




## Checklist for the Peer Reviewers
